recipe-client-addon/bin/tc/package.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -eu
+
+# mach wants this
+export SHELL=$(which bash)
+
+# Fetches build results, and creates ./gecko-dev-master/
+echo 'Downloading build result'
+curl --location --fail --silent --show-error "$BUILD_RESULT" | tar xz
+
+echo 'Setting up environment'
+pushd gecko-dev-master
+source /root/.cargo/env
+python2.7 ./python/mozboot/bin/bootstrap.py --no-interactive --application-choice=browser
+source /root/.cargo/env
+
+echo 'Packaging Firefox'
+./mach package
```

Comment:
Sure, if we can find a good way to determine the name of that directory. I wouldn't consider it a blocker though.